src/services/zapper.ts

```diff
@@ -69,12 +69,12 @@ export class ZapperService extends Service {
   async balances<T extends Address>(addresses: T[] | T): Promise<BalancesMap<T> | Balance[]>;
 
   async balances<T extends Address>(addresses: T[] | T): Promise<BalancesMap<T> | Balance[]> {
-    const url = "https://api.zapper.fi/v1/protocols/tokens/balances";
+    const url = "https://api.zapper.fi/v2/apps/tokens/balances";
     const params = new URLSearchParams({
       "addresses[]": Array.isArray(addresses) ? addresses.join() : addresses,
       api_key: this.ctx.zapper,
     });
-    const balances = await fetch(`${url}?${params}`)
+    const { balances } = await fetch(`${url}?${params}`)
```

Comment:
As far as I could see, this endpoint is the only one that has changed the structure of the response. Once we add [Zod](https://github.com/colinhacks/zod) it will be easier to see if some response is different from what we expect.